.gitignore

```diff
@@ -13,3 +13,6 @@ settings.db
 
 # C API tests (ignore everything but source files)
 *.dSYM
+
+#ignore jetbrains ide folders
+.idea
```

Comment:
I think we specifically removed IDE specific folders in the git ignore for some reason

Reply:
In rustRover While I was trying to commit, it accidentally added .idea to stage so I had to stash that. So should I remove that?

Reply:
yes please..  IDE and OS specific files should be added to your own gitignore instead of doing this in the project's.

Reply:
Understood, I have reverted.